setup.py

```diff
@@ -163,8 +213,12 @@ def run(self):
     zip_safe=False,
     ext_modules=[cmake_extension('arbor')],
     cmdclass={
-        'build_ext': cmake_build,
-        'install':   install_command,
+        'build_ext':   cmake_build,
+        'install':     install_command,
+        'bdist_wheel': bdist_wheel_command,
+    } if WHEEL_INSTALLED else {
```

Comment:
Maybe store `cmdclass` in a variable set in the first if/else above?

Reply:
OK

Reply:
Turns out this creates a problem: https://github.com/brenthuisman/arbor/runs/2031776731?check_suite_focus=true